firmware/application/ui_navigation.cpp

```diff
@@ -302,6 +311,18 @@ void SystemStatusView::on_camera() {
 	}
 }
 
+void SystemStatusView::on_clk() {
+	bool v = portapack::persistent_memory::clkout_enabled();
```

Comment:
This line could be changed into: 

`bool v = !portapack::persistent_memory::clkout_enabled();`

and then you would not need the:

```
 	if(v) {
 		v = false;
 	} else {
 		v = true;
 	}
 ```